_posts/2017-12-01-ome-files-0-5-0.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Release of OME Files C++ 0.5.0
+intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.5.0
+---
+OME Files is a reference implementation of the OME Data Model and
+OME-TIFF file format for the storage and interchange of biological
+imaging data and metadata.  It provides support for:
+
+- the OME Data Model (model objects, metadata store, XSL transforms
+  and XML validation)
+- OME-TIFF reading
+- OME-TIFF writing
+
+The main change in this release is the re-addition of compatibility
+code to permit building on CentOS6 and RHEL6 with EPEL.  Other work
+includes the switch to C++11 threads and mutexes from the Boost
+equivalents, and preliminary CMake support for building with C++17
+compilers.
+
+**Downloads**
+
+The software release is available at [archived downloads](https://downloads.openmicroscopy.org/ome-files-cpp/0.5.0/). This
```

Comment:
We do not longer have a top-level `index.html` there so the links section probably does not hold. Maybe reduce this section to a single sentence pointing to the raw downloads?